docs/pages/reference/config.mdx

```diff
@@ -3,64 +3,82 @@ title: Teleport Configuration Reference
 description: The detailed guide and reference documentation for configuring Teleport for SSH and Kubernetes access.
 ---
 
-## teleport.yaml
+Teleport uses the YAML file format for configuration. A full configuration
+reference file is shown below. This provides comments and all available options
+for `teleport.yaml`. 
 
-Teleport uses the YAML file format for configuration. A full configuration reference
-file is shown below, this provides comments and all available options for `teleport.yaml`
-By default, it is stored in `/etc/teleport.yaml`.
+By default, the `teleport` process reads its configuration from
+`/etc/teleport.yaml`.
 
-<Admonition type="danger" title="Before using this reference">
-<Tabs>
-<TabItem scope={["oss", "enterprise"]} label="Self-Hosted">
+## Before using this reference
 
-- **Do not use this example configuration in production.**
+<Notice type="danger">
 
-  You must edit your configuration file to meet the needs of your environment.
-  Using a copy of the reference configuration will have unintended effects. To
-  create a configuration file that you can use as a starting point, run the
-  following command:
+Do not use this example configuration in production.
+
+</Notice>
+
+You must edit your configuration file to meet the needs of your environment.
+Using a copy of the reference configuration will have unintended effects. To
+create a configuration file that you can use as a starting point, run the
+following command:
 
   ```code
   $ teleport configure -o file
```

Comment:
Might be worth mentioning the other configure commands as well (teleport node configure, teleport db configure, etc)